e2e/mobile/mobile-learn.spec.ts

```diff
@@ -31,7 +31,26 @@ const publicSB = orderedSuperBlockInfo
   .filter(sb => sb.public === true && !nonEditorSB.includes(sb.dashedName))
   .map(sb => sb.dashedName);
 
-const typedCurriculum = currData as Curriculum;
+const incompatible = [
+  'certifications',
+  'a2-english-for-developers',
+  'b1-english-for-developers'
+];
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const removeNonCompatibleSuperblocks = (currData: any) => {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const copy = currData;
+
+  for (const superBlock of incompatible) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    delete copy[superBlock];
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+  return copy;
+};
+
+const typedCurriculum = removeNonCompatibleSuperblocks(currData) as Curriculum;
```

Comment:
```suggestion
const removeNonCompatibleSuperblocks = (currData: Curriculum): Curriculum => {
  const copy = currData;

  for (const superBlock of incompatible) {
    delete copy[superBlock];
  }
  return copy;
};

const typedCurriculum = removeNonCompatibleSuperblocks(currData as Curriculum);
```

If we are going to typecast anyway, why not avoid the eslint issues?